reactor-core/src/main/java/reactor/core/publisher/FluxOnBackpressureBufferStrategy.java

```diff
@@ -66,6 +66,12 @@ public int getPrefetch() {
 		return Integer.MAX_VALUE;
 	}
 
+	@Override
+	public Object scanUnsafe(Attr key) {
```

Comment:
`FluxOnBackpressureBuffer` isn't covered at all

Reply:
Fixed in commit [26552f6](https://github.com/reactor/reactor-core/pull/2123/commits/26552f67f2cfd7c10e59e60f3b905504c55b6e1d)